src/core/scene/a-scene.js

```diff
@@ -65,11 +65,30 @@ module.exports = registerElement('a-scene', {
           this.setupRenderer();
           this.resize();
         });
+        this.addFullScreenStyles();
         initPostMessageAPI(this);
       },
       writable: true
     },
 
+    addFullScreenStyles: {
+      value: function () {
+        var htmlEl = document.querySelector('html');
+        htmlEl.classList.add('a-html');
+        document.body.classList.add('a-body');
+        this.classList.add('fullscreen');
```

Comment:
I think the collisions chances are minimal and it's already the established pattern. Not worth to go and change all class names now.